src/FdTable.cc

```diff
@@ -122,16 +122,21 @@ FileMonitor* FdTable::get_monitor(int fd) {
   return it->second.get();
 }
 
-static bool is_fd_monitored_in_any_task(AddressSpace* vm, int fd) {
+static enum syscallbuf_fd_classes join_fd_classes_over_tasks(AddressSpace* vm, int fd) {
```

Comment:
Don't need 'enum' keyword here. Also I think the enum name should be CamelCase

Reply:
Dropped the keyword, but kept the name, since it matches the convention in the header file.